### PR TITLE
[concept] wrap multipart/encrypted into multipart/alternative

### PR DESCRIPTION
**Pull request by rrrooommmaaa**
#3297

### CODE REVIEW

File: extension/js/common/core/mime.ts [hunk 1]
```diff
@@ -197,26 +197,40 @@ export class Mime {
 
   public static encode = async (body: SendableMsgBody, headers: RichHeaders, attachments: Attachment[] = [], type?: MimeEncodeType): Promise<string> => {
```
Comment (tomholub) [inline extension/js/common/core/mime.ts:198]: I'm hesitant to merge this type of change if it ends up being used for regular pgp/mime messages. If we only used it for drafts, that would be ok, but I'd prefer not to alter with outgoing messages unless we have to.

Is there some benefit to wrapping outgoing messages this way that I overlooked?

Comment (rrrooommmaaa) [reply extension/js/common/core/mime.ts:198]: No, there is no immediate benefit. This was done as part of experiment, so I wanted to save this code.

Comment (tomholub) [reply extension/js/common/core/mime.ts:198]: Got it, I'll leave this open

**Comment by tomholub** [comment]
Doing a PR cleanup - will close this. Thanks!

**Pull request closed without merging**